app/src/test/java/com/developersbreach/composeactors/konsist/PreviewTest.kt

```diff
@@ -0,0 +1,28 @@
+package com.developersbreach.composeactors.konsist
+
+import com.lemonappdev.konsist.api.Konsist
+import com.lemonappdev.konsist.api.verify.assertFalse
+import org.junit.Test
+
+class PreviewTest {
+
+    @Test
+    fun `no direct usage of androidx compose Preview should be allowed except PreviewLightDark`() {
+        val previewAnnotation = "androidx.compose.ui.tooling.preview.Preview"
+        val excludePaths = arrayOf(
+                "compose-actors/app/src/main/java/com/developersbreach/composeactors/annotations/PreviewLightDark.kt"
+            )
+        Konsist.scopeFromProject()
```

Comment:
You should not create duplications, this functionality in exists as you already know in different function in other test class.
Now since that you know it can be reused, you need to lift out that functionality and reuse it rather than declaring again.
I've placed in same test source package in new file `KonsistUtils` and renamed it to `checkNoDirectUsageExceptAllowed` for obvious reasons.